api/v1alpha2/types.go

```diff
@@ -89,6 +89,11 @@ type NetworkDeviceSpec struct {
 	// will be connected.
 	NetworkName string `json:"networkName"`
 
+	// SetName can be used to give the device a more specific/desirable/nicer
+	// name than the default from udev’s ifnames
+	// +optional
+	SetName string `json:"SetName,omitempty"`
```

Comment:
re: the JSON comment - `s/SetName/setName/`

Reply:
renamed